package.json

```diff
@@ -187,4 +187,4 @@
             "last 1 chrome version"
         ]
     }
-}
+}
```

Comment:
Man don't bother about trailing newlines, focus on the code.
I think I probably removed some boring eslint rules that were making us lose time because we saved the code and then got an error because of some rule that doesn't do any good. It's just styling. We should focus on rules that could catch errors or make our lives easier. 👍🏽